src/@next/components/atoms/DemoBanner/DemoBanner.tsx

```diff
@@ -1,9 +1,7 @@
+import { mediumScreen } from "@styles/constants";
```

Comment:
I believe the placement was correct before, storefront seems to be using the following convention:
```
<module imports>
// newline
<absolute imports>
// newline
<relative imports>
```
The eslint config should enforce that, but we can deal with that later.

Reply:
My vscode config got wild 🙈 ...